collector/src/main/java/org/apache/hertzbeat/collector/collect/nginx/NginxCollectImpl.java

```diff
@@ -139,7 +139,7 @@ private void validateParams(Metrics metrics) throws Exception {
         
         String url = nginxProtocol.getUrl();
         
-        if (StringUtils.isEmpty(url) || !url.startsWith(RIGHT_DASH)) {
+        if (url == null || StringUtils.isEmpty(url) || !url.startsWith(RIGHT_DASH)) {
```

Comment:
![image](https://github.com/apache/hertzbeat/assets/15684156/3a7039de-5c0c-44d9-9aaa-b10f7812fade)

The method `StringUtils.isEmpty(url)` has already checks whether string is null or empty.

So maybe we don't need this check `url == null`.